packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -389,6 +400,24 @@ function InfoTooltip({ orderer }: { orderer: string }) {
   );
 }
 
+function ExpandedOrderView({ row }: { row: any }) {
```

Comment:
For this case the data being passed is the `row` from Carbon's DataTable that's why i passed the row and handled the logic in this function. The type `any` was because the structure is mixed.